cartridges/int_alma/cartridge/scripts/helpers/almaCheckoutHelper.js

```diff
@@ -159,6 +165,44 @@ function isFragmentActivated() {
     return Site.getCurrent().getCustomPreferenceValue('ALMA_Fragment_Payment');
 }
 
+/**
+ * Return true if plan is activated
+ * @param {Object} paymentMethod payment method
+ * @param {Object} plan plan
+ * @returns {boolean} payment method ID
+ */
+function planIsActivated(paymentMethod, plan) {
+    var almaActivated = paymentMethod.getCustom().almaActivated.split(' | ');
+    var isActivated = false;
+
+    almaActivated.forEach(function (element) {
+        if (element.includes(plan.installments_count || plan.deferred_days)) {
+            isActivated = true;
+        }
+    });
+
+    return isActivated;
```

Comment:
you could use [some](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/some)

```
    return almaActivated.some(function (element) {
        return element.includes(plan.installments_count || plan.deferred_days);
    });
```

I don't think that this line is doing what you want :
```
        if (element.includes(plan.installments_count || plan.deferred_days)) {
```
I think what you want to achieve is 
```
        if (element.includes(plan.installments_count) || element.includes(plan.deferred_days))) {
```

So in the end :

```suggestion
    return almaActivated.some(function (element) {
        return element.includes(plan.installments_count) || element.includes(plan.deferred_days));
    });
```